src/app/CommandHandler.cpp

```diff
@@ -186,7 +181,7 @@ void CommandHandler::DecrementHoldOff()
         return;
     }
 
-    if (!mExchangeCtx->IsGroupExchangeContext() && !mSentStatusResponse)
```

Comment:
Isn't this now going to cause us to send two messages: the IM engine sending a status response (see [this](https://github.com/project-chip/connectedhomeip/pull/19356/files#diff-09d4e6f8f568b8ae1f60c465f4f2c89e3fb4e92d92ac2308fae80036b2430a1dR594)), and this here sending a response?